tools/babel-preset/lib/builder.js

```diff
@@ -19,7 +19,7 @@ const defaultPlugins = ['@babel/proposal-class-properties', '@babel/proposal-obj
 
 const typescriptRequirements = {
   presets: ['@babel/typescript'],
-  plugins: [],
+  plugins: ['@babel/proposal-nullish-coalescing-operator', '@babel/proposal-optional-chaining'],
```

Comment:
Is this still required since it ships by default in TypeScript 3.7? I am confused about this because i don't use babel to parse TypeScript.

Reply:
Typescript is only helpful in the IDE, you also need Babel being setup so the compiler also is able to understand the new features.